### ISSUE TITLE
<Warning: Could not be resolved, but is possibly hardcoded in the engine>

### ISSUE THREAD

**Issue by nightmare2004-nexusmods** [TES5Edit/TES5Edit#1399]
**What is the version of xEdit you are using?**
SSEEdit 4.1.5f (094C4648) starting session 2024-12-16 14:08:17

**Describe the bug**
I'm working on an outfit form to distribute using SPID and when I attempt to add the items to the outfit form, it will show the name and id for the item, I will then click on it and it will display properly in SSE Edit but when I click off or press enter it will then change to an error saying <Warning: Could not be resolved, but is possibly hardcoded in the engine> and changes the first 5 digits of the id to 0's, for example (it's for an Skyrim AE Mod called Senecio Armor): 
Senecio_Steel_Boots "Steel Senecio Boots" [ARMO:FE06840D]
...becomes...
[0000040D] <Warning: Could not be resolved, but is possibly hardcoded in the engine>

**To reproduce**
1. Load Plugins in SSEEdit
2. Right click left plane
3. Scroll down to, and click, Create New File
4. Select from drop down the third option (Template ESL)
5. Name new file, press enter
6. Right click new file, selected Add and choose Outfit entry
7. Right click new Outfit entry, click add and select Outfit, press enter to accept form id.
8. Right click Outfit entry again and select Add Masters, and select SenecioArmor.esp and press enter.
9. Left click Outfit entry and move mouse to right plane, right click Items entry and select Add.
10. Double click the new null entry, select from dropdown menu "Senecio_Steel_Boots "Steel Senecio Boots" [ARMO:FE06A40D] or really any entry from it"
11. Click off or press enter
12. See that it's changed to "[0000040D] <Warning: Could not be resolved, but is possibly hardcoded in the engine>"
13. Cry.

**Expected behavior**
What I expected to happen was that the item entry (for the outfit form) would stay as Senecio_Steel_Boots "Steel Senecio Boots" [ARMO:FE06840D], and I would then move on to add the other two item entries to complete the [outfit.](![ArmorErrpr]

**Screenshots**
(https://github.com/user-attachments/assets/e0e386c5-48e8-48d9-8c7a-f4083f76aacd))
The first item entry is what it SHOULD look like and is only for display purposes, when I press enter or click off it then changes the words to the second entry, they are the same item.

**Additional context**
I have tested other armor mods from the same author and they work fine, I've brought up to them and as of now all they suggested was to update SSE Edit, but I am using the most up to date version, so I offered to come here to learn at least what this error is trying to tell me so I can then pass it on to them.

Update: Played around with it a bit, if I add the armor as an armor entry into the new esl THEN it works. I'm not sure if I'm allowed to do that though or if that creates other issues. I hope that can help.

**Comment by robertgk2017** [comment]
plugins that define new records below formid 800 must be Rcord HEDR version 1.71.

plugins Referencing such records must also be HEDR 1.71.

what you are witnessing is xEdit working as intended and desired.

also, update to the current version of xEdit 4.1.5k from our builds channel

**Comment by nightmare2004-nexusmods** [comment]
Hmm thank you for your response. Would that be the Version entry within the HEDR - Header field of the esp? If so, it says it is 1.710000, is that not the correct way of it being stated, or is that the wrong field?

Also, where would the builds channel be? I'm unable to find anything like that here.

Edit: Soo, I found the XEdit Discord, found a build channel and downloaded what you said but, that's for Fallout 4 isn't it? I'm modding Skyrim AE, how do I use xEdit to mod Skyrim? (I thought that's what SSEEdit was for)?

**Comment by nerwus3456** [comment]
Christ, xEdit is the very same tool for every supported game. All you have to do is rename the xEdit.exe to SSEEdit.exe to use it for Skyrim, or make a shortcut and add -sse

**Comment by nightmare2004-nexusmods** [comment]
Ah, I didn't know that, thank you.

Edit: Well, I tried it out in xEdit and it's still doing it. So, I'm not sure what else I can provide.

Edit 2 (Final): So I decided to give something a try. I learned that the error doesn't happen if I add the items as an armor entry into my esl first (overwrite as new entry), then it works. I then removed the armor entries from my esl once the items were placed in the outfit entry, and the error stayed gone. I clicked off, no error, saved, no error, closed and tested in game and the spid works, the armor is being distributed. I have no idea what caused that, I have no idea why my solution "worked" (I would be surprised if there wasn't some issue with what I did down the road), it "just works". I am on the most up to date version of the game, and xedit and I believe all of the mods are in version 1.71, so I don't rightly know what's going on anymore.

So I guess the issue is "fixed", although I would like to learn why this was happening for my own curiosity and learning, and I hope these posts and updates are of help. I'll keep an eye on this topic for the next week in case you have any questions, I'd like to help if I can be of any help. Thank you for your help.
